pages/common/for.md

```diff
@@ -0,0 +1,11 @@
+# for
+
+> Shell loop over parameters
+
+- Perform an action with different arguments.
+
+`for argument in 1 2 3; do {{cmd $argument}}; done`
+
+- Perform an action in every directory.
+
+`for d in *; do (cd $d; make test); done`
```

Comment:
Ah, this was changed before my comment above, please disregard. @denis-sokolov's improvement was a better one than my suggestion :D